### ISSUE TITLE
Spell cost indicator vanishes progress bar

### ISSUE THREAD

**Issue by nbf77** [WeakAuras/WeakAuras2#2510]
## Describe the bug
I am on my marksman hunter and casting aimed shots back to back. i created a progress bar for focus.
when i'm pressing aimed shot, than a part of the progress bar become white to show my focus after the cast.
However, this white bar vanishs randomly. I'll try to explain this more detailed below.

**Do you have an error log of what happened?**
No :-(

**Did you try having WeakAuras as the only enabled addon and everything else (especially something like ElvUI) disabled?**
yes, same issue

**Which version of WeakAuras are you using?**
2.18.4

**Are you on World of Warcraft Classic or Retail?**
Retail

**Was it working in a previous version? If yes, which was the last good one?**
I never hat issues with that. last time i used that is long ago, though.

### Additional Info
I tried to reproduce, this is what I could notice:
- does not happen in combat
- seems to happen at any point of the cast time
- sometimes it does not happen for 1-2 casts
- pressing a button once or multiple times doesnt influence it

WA String:

> !WA:2!1vv3UnXru4SYcQQLsf42slvi1uKaXnjLe(PqBPuVgBSrgB31oeafj7z3z8UtZ6z2oZSojCtf(kU2Tpb(AUYpbv5fOJC7tquvFaYtqpZSBsaeDLs2ZC2ZpZ57CoF2PwPrLWLWVA9zYeKyNTOyv0NEGvU9WHsI6PoI9YKCe7Nju)p2DWsPL(1)CoIfeXfD4uMYVs1w9Q6TW6QhxHuuoZj58V15hZXe3YB2RTlnGZIMOe0WqIq(rxvKl(VUQ9ti(sW(u5HYuFYyct1nD4q6EZ7xPC3E972RSxppREVo8DjIj4uHn8Lw)GujPVmIVBfUunyI5ukJQgiszwhCrPk(0eJxM8CMjYesCCdSS4j5QJGa5YRBNQnB6XqJiYIUMy4NeJ2NiwyI5jbyWmmXhUC9azr9Qn7uBZMttz5ftXKZJcu0XKEzNnL)RxAMP27hWJ5IhvaEocqpIGHIFcGeqvuzQKep0cQ6Z6Qi7PIM4JevSou)(hwY(u)()1VzE(DNcWDqcWfGbKO3S19mNzqr5rcHGkF0f0lphbgbHQp83RUqsjtSb0cH57(0mxkm1QZ8VlFLKKZ9gwKLFhhNchMP90lzYhB1yW2rqJiWof5wgoEK9daEsWwLVCr2iZZYhPoW(9FovQOd3xF2zcZGcPhO0Tv7wv1llGkWyZmRHNcAz3ZHCMk5C1e0xSYpLIWacGwPxV3QWYZ0LwklvJPsQFmzWc7PD5c8wcuYKTYfwCsynDu3wCgjheEJr9dA0QvvV(zd8jF1jig4NaQ)(j9v0rWqIGeqnn0cjx89ytMGNXu8PPTl9fKBKds59eNKV8)lf49HbuAq0R(e9Qf0RD5RW0FTTfPVU(d1RR3qFJxQVPJ(w6ZQVT(B0lRVJ(Uw8t)T6Vt)9xAj9pOVV(ErA35zfLxJhwVN(h1L1p8I66oVNBolxWLbGZy9YtHoKpaFWQr2RCOAdyiBEUgjuvN5G8dbNoiLR54MYXMtWHKKZ39xsrcYk1sJJxzRiQICC85zWIFeHggPUNBmhHRn1SxgeJKYbE2xx1xszHXe)6BA6tEJsJv0IUWcFWv1pVGE7VyGR5EDb92WYVzRXnMWWMD0nk)GhKCoiGLXMrtcMGFmAVOltXE147Kkxat6qf2oXq6ilofXOJSeq32diUeQAhqqssxLGWcvrFaa7nxCmhvFdTHqsaoaSCIXoZKMUP7ieLvtp0ATouhPP6M(dPmQm6DvFOCeNRI6i4HcIug9U3vklAMWU2B5LY36NZhteahwqmnzWuljKKIjE2wU7l48ro6pBWCluupdAfMgFQGmRA84nBSslOX7d(nKgwmJOPx23tUwddh2quaz7kiyzMf6IeBVzJvp90QDT)sG1TOzkEW4m6Up35UVopPumMW8Av9jv9okBiQMaiGnfHF3kEvR2A(qJcazHn9cE7AOu(NsEO4KiuHnsP45LVwuO6zbTVZdhkOmtkirz)0wLt4oMZfuGO32oMvVTxJN3UvVYnNzAj0SwQWWBdc3wtkQ3POl0gufpYhfStOGNYWNqg2PW1x7wLK(BS263zTBwA8F)0)7

/edit 
added WA string

**Comment by mrbuds** [comment]
Do you have a video clip showing this ? It works fine for me.

**Comment by nbf77** [comment]
> Do you have a video clip showing this ? It works fine for me.

Hmm ... I'll make a short clip tomorrow. I will also try to do it naked (to check if it's a trinket or something?!) and perhaps change all talents. But it's probably a strange problem in my settings? I'll create a new hunter and check it out there as well.

I'll keep you updated!

**Comment by InfusOnWoW** [comment]
I can't reproduce it either. If you can figure out what additional things are needed, just respond to the ticket and we'll take another look.

**Comment by nbf77** [comment]
Hey,
finally I had some time testing it.

I figured out when this happens: It always happens when the swing timer finishes (casting an auto-shot) during an aimed shot cast (see my animated gif!). 
What I did to try to remove that bug:
- I deactivated ALL addons
- I changed every single talent
- I did it naked (except for a bow)
- I deactivated all weakauras (set to load = never)

![Uploading aimed_shot_wa.gif…]()

Is anyone able to reproduce this?

Thanks!
nbf77
